Sources/PointFree/Database.swift

```diff
@@ -201,3 +149,69 @@ public func migrate() -> EitherIO<Error, Prelude.Unit> {
     )))
     .map(const(unit))
 }
+
+#if os(iOS)
```

Comment:
all of this can be removed since we dont run tests for this repo on ios anymore! I've done it in my branch, but I'll resolve the conflicts.